mw-webapp/src/service/MentorCommentsService.ts

```diff
@@ -0,0 +1,23 @@
+import {collection, getDocs} from "firebase/firestore";
+import {db} from "src/firebase";
+import {MentorCommentDTO} from "src/model/firebaseCollection/MentorCommentDTO";
+import {querySnapshotToDTOConverter} from "src/service/converter/querySnapshotToDTOConverter";
+
+const PATH_TO_MENTOR_COMMENTS_COLLECTION = "mentorComments";
+
+/**
+ * MentorComments requests: {@link getMentorComments}
```

Comment:
1. What do yo umean requests? Dont use synonims
2. what is  getMentorComments? How does it related to this service?

Reply:
I followed comment style from other services after their refactoring and moving to fireStore, i thought that's how they supposed to be

Reply:
@link is a tool to ger reference in docs, that's all. You can tell everyone: if you can't add any additional info in the doc - that does not mean you should just type something using synonyms. Docs should simplify the project, not complexify)

Reply:
So, should i change doc in other services too? Changed comment in this one, please check.

Reply:
It is up to you. Usually - yes if it is simle